src/library/Uno.Themes.WinUI.Markup/Theme/CheckBox.cs

```diff
@@ -1,227 +1,230 @@
-﻿using Microsoft.UI.Text;
+using System;
+using Windows.UI;
+using Microsoft.UI.Text;
 using Microsoft.UI.Xaml;
 using Microsoft.UI.Xaml.Media;
 using Uno.Extensions.Markup;
 using Uno.Extensions.Markup.Internals;
-using MUXCornerRadius = Microsoft.UI.Xaml.CornerRadius;
+
+using ResourceKeyDefinitionAttribute = Uno.Themes.WinUI.Markup.ResourceKeyDefinitionAttribute;
 
 namespace Uno.Themes.Markup
 {
 	public static partial class Theme
 	{
-		public static class CheckBox
+		public static partial class CheckBox
 		{
-			[ResourceKeyDefinition(typeof(double), "CheckBoxFocusAreaSize")]
-			public static ThemeResourceKey<double> FocusAreaSize => new("CheckBoxFocusAreaSize");
-
-			public static class Resources
+			public static partial class Resources
 			{
-				public static class Background
+				public static partial class CheckArea
+				{
+					[ResourceKeyDefinition(typeof(Thickness), "CheckBoxCheckAreaBorderThickness")]
+					public static ThemeResourceKey<Thickness> BorderThickness => new("CheckBoxCheckAreaBorderThickness");
+
+					[ResourceKeyDefinition(typeof(CornerRadius), "CheckBoxCheckAreaCornerRadius")]
+					public static ThemeResourceKey<CornerRadius> CornerRadius => new("CheckBoxCheckAreaCornerRadius");
+
+					[ResourceKeyDefinition(typeof(Thickness), "CheckBoxCheckAreaPadding")]
+					public static ThemeResourceKey<Thickness> Padding => new("CheckBoxCheckAreaPadding");
+				}
+
+				public static readonly CheckBackgroundFillUncheckedVSG CheckBackgroundFillUnchecked = new();
```

Comment:
Naming seems off, includes Unchecked in the name but the states include the Checked and Indeterminate states as well. 

If we could somehow keep it grouped as Foreground/Background/BorderBrush/Glyph that'd be awesome